client/src/components/nodeStatusChart.tsx

```diff
@@ -2,8 +2,9 @@ import _ from 'lodash';
 import React from 'react';
 import Chart from './chart';
 import LoadingChart from './loadingChart';
+import {TODO} from "../utils/types";
```

Comment:
Same thing here. There's a `Node` type now. Again, it's just a stub and needs to have fields added to it, but feel free to add whatever fields are needed until it compiles.